Synchronize opacity and compositeOperation of TiledImages in navigator

seanich opened the pull request:
Currently the `TiledImage`s in the navigator are not updated to reflect changes to the `opacity` and `compositeOperation` of `TiledImage`s in the main viewer. This PR fixes that by adding two events, `opacity-change` and `composite-operation-change`, to `TiledImage` and listening to those events in the navigator.

iangilman commented on src/tiledimage.js:
```diff
@@ -771,10 +771,12 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
     setOpacity: function(opacity) {
         this.opacity = opacity;
         this._needsDraw = true;
+        this.raiseEvent('opacity-change');
```
Please add documentation for this event (similar to `clip-change` and `bounds-change`), and the other new event. 

Also, it might be good to include the new opacity/composite operation in the event, similar to the `fully-loaded-change` event.

seanich commented:
I've added the documentation and the extra event parameters. I've also added an early exit condition to `setOpacity` and `setCompositeOperation` so that they don't raise and event or cause a redraw when nothing has been changed.

Let me know if you need anything else :)

iangilman commented:
@seanich Beautiful! Looks perfect, thank you :)
